ui/src/components/sidebar/connectionStats.tsx

```diff
@@ -66,15 +29,13 @@ export default function ConnectionStatsSidebar() {
   useInterval(function collectWebRTCStats() {
     (async () => {
       if (!mediaStream) return;
-      const videoTrack = mediaStream.getVideoTracks()[0];
```

Comment:
Is this guard being removed intentional? Seems odd that we shouldn't be ensuring we have a `videoTrack` like before?